skrf/vi/vna/hp/__init__.py

```diff
@@ -1 +1,10 @@
+"""
+.. module:: skrf.vi.vna.hp
+===========================================
+HP VNAs (:mod:`skrf.vi.vna.hp`)
+===========================================
+
+From the age when HP made expensive test equipment rather than expensive printer ink.
```

Comment:
The comment is OK to me ;  I let others say something eventually ;)

Reply:
I love it!

Reply:
Well I had to find something to say about HP in general, lol. I'm afraid this will be hidden for now -- it looks like Sphinx doesn't generate module docs unless they are linked elsewhere and given that we have only 1.3 VNAs per manufacturer at the moment I think it's probably best to keep the skrf.vi.vna page as a list of classes right now. One day, though, there will be enough VNAs to break out the categories, and on that day I hope to contribute to the archaeological record about HP :)